packages/tx/src/eip1559Transaction.ts

```diff
@@ -152,6 +153,8 @@ export default class FeeMarketEIP1559Transaction extends BaseTransaction<FeeMark
       s,
     ] = values
 
+    validateNoLeadingZeroes({ nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, value, v, r, s })
```

Comment:
Hmm interesting great catch! Was this found by the new  ethereum/tests?

Reply:
Yes.